olioapps/typescript-starter-lab
# Add jest tests for all methods, add immutability to methods

**Ticket**
Not applicable

**What does this PR do**
- Immutability has been added to all methods
- Writes Jest tests for all methods
- Removes original console.log tests

**Media**
![image](https://user-images.githubusercontent.com/7016167/173118499-5c83461d-d3ec-490f-8970-1b0e9844eebe.png)

**How to test**
To test, clone the repository to your computer and navigate to the project with the terminal. Download all necessary packages with **_yarn install_** and when ready, run **_yarn run test_** to execute the application. You should see several tests and the results of these tests.
